pkg/services/dashboards/database/database_test.go

```diff
@@ -93,7 +93,7 @@ func TestIntegrationDashboardDataAccess(t *testing.T) {
 		setup()
 		query := dashboards.GetDashboardQuery{
 			Title:    util.Pointer("test dash 23"),
-			FolderID: &savedFolder.ID,
+			FolderID: &savedFolder.ID, // nolint:staticcheck
```

Comment:
I assume it works to add `// nolint:staticcheck` to the line in question instead of above.

Reply:
yes!